### ISSUE TITLE
weights file of SegResNet only includes: key(s) in state_dict: "net", "optimizer", "scheduler", "epoch".

### ISSUE THREAD

**Issue by meeselizabeth** [MrGiovanni/SuPreM#5]
I am trying to load your SegResNet weights in the SegResNet model, however I get the error: 

Error(s) in loading state_dict for SegResNet:
	Missing key(s) in state_dict: "convInit.conv.weight", "down_layers.0.1.norm1.weight", "down_layers.0.1.norm1.bias", "down_layers.0.1.norm2.weight", "down_layers.0.1.norm2.bias", "down_layers.0.1.conv1.conv.weight", "down_layers.0.1.conv2.conv.weight", "down_layers.1.0.conv.weight", "down_layers.1.1.norm1.weight", "down_layers.1.1.norm1.bias", "down_layers.1.1.norm2.weight", "down_layers.1.1.norm2.bias", "down_layers.1.1.conv1.conv.weight", "down_layers.1.1.conv2.conv.weight", "down_layers.1.2.norm1.weight", "down_layers.1.2.norm1.bias", "down_layers.1.2.norm2.weight", "down_layers.1.2.norm2.bias", "down_layers.1.2.conv1.conv.weight", "down_layers.1.2.conv2.conv.weight", "down_layers.2.0.conv.weight", "down_layers.2.1.norm1.weight", "down_layers.2.1.norm1.bias", "down_layers.2.1.norm2.weight", "down_layers.2.1.norm2.bias", "down_layers.2.1.conv1.conv.weight", "down_layers.2.1.conv2.conv.weight", "down_layers.2.2.norm1.weight", "down_layers.2.2.norm1.bias", "down_layers.2.2.norm2.weight", "down_layers.2.2.norm2.bias", "down_layers.2.2.conv1.conv.weight", "down_layers.2.2.conv2.conv.weight", "down_layers.3.0.conv.weight", "down_layers.3.1.norm1.weight", "down_layers.3.1.norm1.bias", "down_layers.3.1.norm2.weight", "down_layers.3.1.norm2.bias", "down_layers.3.1.conv1.conv.weight", "down_layers.3.1.conv2.conv.weight", "down_layers.3.2.norm1.weight", "down_layers.3.2.norm1.bias", "down_layers.3.2.norm2.weight", "down_layers.3.2.norm2.bias", "down_layers.3.2.conv1.conv.weight", "down_layers.3.2.conv2.conv.weight", "down_layers.3.3.norm1.weight", "down_layers.3.3.norm1.bias", "down_layers.3.3.norm2.weight", "down_layers.3.3.norm2.bias", "down_layers.3.3.conv1.conv.weight", "down_layers.3.3.conv2.conv.weight", "down_layers.3.4.norm1.weight", "down_layers.3.4.norm1.bias", "down_layers.3.4.norm2.weight", "down_layers.3.4.norm2.bias", "down_layers.3.4.conv1.conv.weight", "down_layers.3.4.conv2.conv.weight", "up_layers.0.0.norm1.weight", "up_layers.0.0.norm1.bias", "up_layers.0.0.norm2.weight", "up_layers.0.0.norm2.bias", "up_layers.0.0.conv1.conv.weight", "up_layers.0.0.conv2.conv.weight", "up_layers.1.0.norm1.weight", "up_layers.1.0.norm1.bias", "up_layers.1.0.norm2.weight", "up_layers.1.0.norm2.bias", "up_layers.1.0.conv1.conv.weight", "up_layers.1.0.conv2.conv.weight", "up_layers.2.0.norm1.weight", "up_layers.2.0.norm1.bias", "up_layers.2.0.norm2.weight", "up_layers.2.0.norm2.bias", "up_layers.2.0.conv1.conv.weight", "up_layers.2.0.conv2.conv.weight", "up_samples.0.0.conv.weight", "up_samples.1.0.conv.weight", "up_samples.2.0.conv.weight", "conv_final.0.weight", "conv_final.0.bias", "conv_final.2.conv.weight", "conv_final.2.conv.bias". 
	Unexpected key(s) in state_dict: "net", "optimizer", "scheduler", "epoch".

Hopefully you can help me with this!

Best,
Mees

**Comment by WenxuanChelsea** [comment]
Hi @meeselizabeth 

Thanks for your interest in our work!

To load our released SegResNet weights, please follow the example in the code below:
https://github.com/MrGiovanni/SuPreM/blob/56a2aac6d44daec376546773c4ad76c7e573458b/target_applications/totalsegmentator/train.py#L244

Let me know if this works! 

Best,
Wenxuan

**Comment by meeselizabeth** [comment]
Hi @WenxuanChelsea ,

Got it, thank you!

Best,
Mees
